abi/abi.go

```diff
@@ -357,6 +372,15 @@ func Deserialize(data []byte, t Type) (*Value, error) {
 			Type: t,
 			Val:  m,
 		}, nil
+	case PoStProofs:
+		var slice []proofs.PoStProof
+		if err := cbor.DecodeInto(data, &slice); err != nil {
```

Comment:
I don't understand what you're asking for. Are you asking that I propose a modification to `data-structures.md` in the specs repo to include the `proofs.PoStProof` type? 

I do not understand how the PR which you linked to interacts with the code in this PR.

Reply:
that spec says how things in filecoin are to be cbor encoded. this type you are cbor encoding here, postproof, is not in the list of things that that spec says how to cbor encode. it seems like it should be to avoid ambiguity.

Reply:
@laser would be great to have this type in the data-structures spec doc :)